prime-router/settings/STLTs/DE/de-dph.yml

```diff
@@ -0,0 +1,114 @@
+      receivingApplicationName: null
+      receivingApplicationOID: null
+      receivingFacilityName: null
+      receivingFacilityOID: null
+      messageProfileId: null
+      replaceValue: {}
+      replaceValueAwithB: {}
+      reportingFacilityName: null
+      reportingFacilityId: null
+      reportingFacilityIdType: null
+      suppressQstForAoe: false
+      suppressHl7Fields: null
+      suppressAoe: false
+      defaultAoeToUnknown: false
+      replaceUnicodeWithAscii: false
+      useBlankInsteadOfUnknown: null
+      truncateHDNamespaceIds: true
+      truncateHl7Fields: null
+      usePid14ForPatientEmail: false
+      convertTimestampToDateTime: null
+      cliaForOutOfStateTesting: null
+      cliaForSender: {}
+      phoneNumberFormatting: "STANDARD"
+      suppressNonNPI: false
+      processingModeCode: null
+      replaceDiiWithOid: null
+      applyOTCDefault: false
+      useOrderingFacilityName: "STANDARD"
+      valueSetOverrides: {}
+      nameFormat: "STANDARD"
+      receivingOrganization: null
+      convertPositiveDateTimeOffsetToNegative: false
+      stripInvalidCharsRegex: null
+      convertDateTimesToReceiverLocalTime: false
+      useHighPrecisionHeaderDateTimeFormat: false
+      type: "HL7"
+      truncationConfig:
+        truncateHDNamespaceIds: true
+        truncateHl7Fields: []
+        customLengthHl7Fields: {}
+    jurisdictionalFilter:
+      - "(Bundle.entry.resource.ofType(ServiceRequest)[0].requester.resolve().organization.resolve().address.state = 'DE') or (Bundle.entry.resource.ofType(Patient).address.state = 'DE')"
+    qualityFilter: []
+    routingFilter:
+      # Message id is not empty (MSH-10)
+      - "Bundle.identifier.value.exists()"
+      - "Bundle.entry.resource.ofType(Patient).name.family.exists()"
+      - "Bundle.entry.resource.ofType(Patient).name.given.count() > 0"
+      - "Bundle.entry.resource.ofType(Patient).birthDate.exists()"
+      #  Specimen type (SPM-4) is not empty
+      - "Bundle.entry.resource.ofType(Specimen).where(type.empty()).count() = 0"
+      # Patient has at least a street address(PID-11-1), postal code (PID-11-5),phone(PID-13) or email(PID-13-4)
+      - "Bundle.entry.resource.ofType(Patient).where(address.line[0].empty() and address.postalCode.empty() and telecom.where(system = 'phone').value.empty() and telecom.where(system = 'email').value.empty()).count() = 0"
+      - "((Bundle.entry.resource.ofType(Specimen).collection.collectedPeriod.exists() or Bundle.entry.resource.ofType(Specimen).collection.collected.exists())
+            or (Bundle.entry.resource.ofType(ServiceRequest).occurrence.exists() or Bundle.entry.resource.ofType(Observation).effective.exists()))"
+    processingModeFilter: []
+    reverseTheQualityFilter: false
+    conditionFilter:
+      # Accept COVID only
+      - "(%resource.code.coding.extension('https://reportstream.cdc.gov/fhir/StructureDefinition/condition-code').value.where(code in ('840539006')).exists())"
```

Comment:
Same here, you can leave it out if you want.

Reply:
I think it's fine to leave this here. We can update this once we know which conditions they're interested in. Every receiver should have a condition filter to prevent them from receiving unwanted conditions. If SimpleReport starts sending HIV results we don't want those to go to STLTs accidentally if they don't want them.